internal/bminventory/inventory.go

```diff
@@ -86,6 +86,7 @@ type Config struct {
 	ignition.IgnitionConfig
 	AgentDockerImg                  string            `envconfig:"AGENT_DOCKER_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
 	ServiceBaseURL                  string            `envconfig:"SERVICE_BASE_URL"`
+	ImageServiceBaseURL             string            `envconfig:"IMAGE_SERVICE_BASE_URL"`
```

Comment:
should you default to ""?

Reply:
That's the zero value for strings, so that's effectively what I'm doing.